listen.js

```diff
@@ -0,0 +1,33 @@
+const { networks } = require("./networks")
```

Comment:
Lets put this file in the `scripts` folder.  If we want we can add `nodemon`, but I think just putting a command in package.json so you can do something like `npm run listen` would be sufficient too.